aldryn_events/tests/frontend/integration/pages/alteli.js

```diff
@@ -0,0 +1,166 @@
+// ALDRYN TESTS LIBRARY
+// ############################################################################
+'use strict';
+/* global expect, browser, jasmine */
+/* jshint browser: true */
+/* jshint shadow: true */
+
+var aldrynTestsLibrary = {
+    // utility to set browser window size
+    setWindowSize: function (width, height) {
+        browser.driver.manage().window().setSize(
+            width,
+            height
+        );
```

Comment:
Why are you breaking the line here? This would fit comfortably within the 80 column limit (64, in fact, i checked :) ).